stubs/setuptools/setuptools/dep_util.pyi

```diff
@@ -1 +1 @@
-def newer_pairwise_group(sources_groups, targets): ...
+from ._distutils._modified import newer_group as newer_group, newer_pairwise_group as newer_pairwise_group
```

Comment:
This entire module (`stubs/setuptools/setuptools/dep_util.pyi`) is deprecated in favor of `setuptools.modified`
I could keep the defs instead of importing from `_distutils` so I can mark with `@deprecated`